src/modules/reporting/components/reporting-dispute-markets/reporting-dispute-markets.jsx

```diff
@@ -29,6 +29,9 @@ export default class ReportingDisputeMarkets extends Component {
     isForking: PropTypes.bool.isRequired,
     forkEndTime: PropTypes.string,
     currentTime: PropTypes.number.isRequired,
+    isForkingMarketFinalized: PropTypes.bool,
+    forkingMarket: PropTypes.string,
+    updateModal: PropTypes.func.isRequired,
   }
 
```

Comment:
same comment as above

not sure why these proptypes are here. are they used anywhere?